.coderabbit.yaml

```
@@ -0,0 +1,227 @@
+  collapse_walkthrough: false
+  changed_files_summary: true
+  sequence_diagrams: true  # Useful for A2A communication flows
+  assess_linked_issues: true
+  related_issues: true
+  related_prs: true
+  suggested_labels: true
+  auto_apply_labels: true  # Auto-apply relevant labels
+  suggested_reviewers: true
+  auto_assign_reviewers: false
+  poem: true  # Keep it fun!
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)